fe/fe-core/src/main/java/com/starrocks/planner/DeltaLakeScanNode.java

```diff
@@ -94,7 +94,7 @@ private void setupCloudCredential() {
         }
         CatalogConnector connector = GlobalStateMgr.getCurrentState().getConnectorMgr().getConnector(catalog);
         if (connector != null) {
```

Comment:
actually, connector can't be null, just remove if, you can ask @stephen-shelby for confirmation

Reply:
YES, you are right. The connector couldn't be null and has been verified in the analyze phase.

Reply:
fixed.